CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Contributing to api-designer
+
+Do you want to help improving this project?? Here are some instructions to get you started. They are probably not perfect, please let us know if anything feels wrong or incomplete.
```

Comment:
Wrong tense used: "... help improve this project?" The rest of the paragraph is a little awkward but makes sense.